css/style.css

```diff
@@ -1,84 +1,227 @@
-     font-size: 12px;
-     line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+ color: #000;
+ font-size: 12px;
+ line-height: 1.4;
+ font-family: 'Open Sans', sans-serif; 
+}
+button {
+  border-radius: 5px;
+  padding: 8px 35px;
+  text-align: center;
+  margin: 10px;
 }
-
-
 /******************************************
 /* LAYOUT
 /*******************************************/
 
-/* Center the container */
-#container {
-  width: 100%;
-  margin: auto;
+/* Center the Container */
+#container {  
   height: 500px;
-
+  width: 900px;
+  margin: auto;
 }
-
 header {
+  background: url(../images/header_bg.jpg) no-repeat center top fixed rgb(134,141,157);
+  min-height: 500px;
+}
+.header-container {
+  width: 800px;
+  margin: auto;
+  text-align: center;
+  padding: 40px 20px;
+}
+.benefits {
+  background: #f0efef;
+}
+.benefits-container {
+  width: 800px;
+  margin: auto;
+  padding: 40px 40px;
+}
+.testimonial {
+  background: url(../images/testimonial_bg.jpg);
+}
+.testimonial-container {
+  width: 40%;
+  margin: auto;
+  text-align: center;
+  padding: 40px 30px;
+}
+.sign-up {
+  background: #f9e42e;
+  text-align: center;
+}
+.sign-up-containter {
+  width: 40%;
+  margin: auto;
+  padding: 40px 30px;
+}
+footer {
+  background: #000;
+  color: #fff;
+}
+.footer-container {
+  width: 40%;
+  margin: auto;
+  text-align: center;
+  padding: 30px;
+}
+
+/******************************************
+/* HEADER
+/*******************************************/
 
+nav {
+  font-weight: 800;
+  font-size: 24px;
+  color: #fff;
+  padding: 20px 0;
+  text-align: left;
+  margin-left: 20px;
+}
+h1 {
+  font-weight: 600;
+  font-size: 24px;
+  color: #fff;
+  text-align: center;
+  margin: 20px 0;
+}
+#btn-header {
+  font-weight: 800;
+  font-size: 14px;
+  background: #f9e42e;
+  color: #033048;
+  border: 1px solid #f9e42e;
+  margin-top: 30px;
 }
 
-footer {
+/******************************************
+/* SECTION BENEFITS
+/*******************************************/
 
+h2 {
+  font-weight: 600;
+  font-size: 20px;
+  padding-bottom: 10px;
+}
+ul {
+  list-style: disc;
+  list-style-position: inside;
+  font-weight: 300;
+  font-size: 13px;
+  margin-left: 40px;
+}
+.benefits-container p {
+  font-weight: 300;
+  font-size: 13px;
+  padding-bottom: 10px;
 }
 
 /******************************************
-/* ADDITIONAL STYLES
+/* SECTION TESTIMONIAL
 /*******************************************/
 
+blockquote {
+  margin: 0;
+}
+.testimonial blockquote {
+  font-weight: 300;
+  font-size: 18px;
+  font-style: italic;
+  color: #fff;
+}
+.testimonial cite {
+  font-weight: 600;
+  font-size: 18px;
+  font-style: normal;
+  color: #fff;
+}
 
+/******************************************
+/* SECTION SIGN UP
+/*******************************************/
 
+#btn-sign-up {
+  font-weight: 800;
+  font-size: 14px;
```

Comment:
Thanks for letting me know to fix the sign up button font.  Committed new changes to my pull request. 

- Jessica Lewis

> On Feb 27, 2016, at 4:25 PM, ericmasiello <notifications@github.com> wrote:
> 
> In css/style.css:
> 
> >  
> > +#btn-sign-up {
> > +  font-weight: 800;
> > +  font-size: 14px;
> font-size should be 12px
> 
> —
> Reply to this email directly or view it on GitHub.
>